src/dmd/declaration.d

```diff
@@ -1060,24 +1060,49 @@ extern (C++) class VarDeclaration : Declaration
     enum AdrOnStackNone = ~0u;
     uint ctfeAdrOnStack;
 
-    bool isargptr;                  // if parameter that _argptr points to
-    bool ctorinit;                  // it has been initialized in a ctor
-    bool iscatchvar;                // this is the exception object variable in catch() clause
-    bool isowner;                   // this is an Owner, despite it being `scope`
-    bool setInCtorOnly;             // field can only be set in a constructor, as it is const or immutable
+    // `bool` fields that are compacted into bit fields in a string mixin
+    private extern (D) static struct BitFields
+    {
+        bool isargptr;                  // if parameter that _argptr points to
+        bool ctorinit;                  // it has been initialized in a ctor
+        bool iscatchvar;                // this is the exception object variable in catch() clause
+        bool isowner;                   // this is an Owner, despite it being `scope`
+        bool setInCtorOnly;             // field can only be set in a constructor, as it is const or immutable
+
+        // Both these mean the var is not rebindable once assigned,
+        // and the destructor gets run when it goes out of scope
```

Comment:
This is a dead comment.  The "both these" is referring to the `mynew` field for custom allocators.

Reply:
Do you want me to update the comment in this PR?